algorithms/merging/merge.py

```diff
@@ -216,15 +216,22 @@ def merge_and_truncate(params, experiments, reflections):
         logger.debug(out.getvalue())
 
     if params.reporting.wilson_stats:
-        if not intensities.space_group().is_centric():
-            wilson_scaling = data_statistics.wilson_scaling(
-                miller_array=intensities, n_residues=params.n_residues
-            )  # XXX default n_residues?
-            # Divert output through logger - do with StringIO rather than
-            # info_handle else get way too much whitespace in output.
-            out = StringIO()
-            wilson_scaling.show(out=out)
-            logger.info(out.getvalue())
+        try:
+            if not intensities.space_group().is_centric():
+                wilson_scaling = data_statistics.wilson_scaling(
+                    miller_array=intensities, n_residues=params.n_residues
+                )  # XXX default n_residues?
+                # Divert output through logger - do with StringIO rather than
+                # info_handle else get way too much whitespace in output.
+                out = StringIO()
+                wilson_scaling.show(out=out)
+                logger.info(out.getvalue())
+        except RuntimeError:
+            logger.error(
+                "\n"
+                "Error encountered during Wilson statistics calculation:\n"
+                "Perhaps there are too few unique reflections.\n"
```

Comment:
```suggestion
                "Perhaps there are too few unique reflections.\n",
                exc_info=True
```
pass on trace information